examples/skywalking/docker-compose.yaml

```diff
@@ -77,6 +77,14 @@ services:
     environment:
       discovery.type: single-node
       ingest.geoip.downloader.enabled: "false"
+
+      # This allows ES to run in a low-resource environment, dont do this at home!
```

Comment:
Is this needed just for our CI execution? If so, it may make sense to add a guard here to set these vars when run under our-CI.
This is just to avoid people copy-pasting it to their docker-compose scripts or copying the env vars from that docker.

Reply:
hmm, yep - i have those messages there for that reason - but dont really want to add this - its not for production

having said that i think its safe here - and probably a good idea - the sandbox was refusing to start because it only had ~5GB spare which is more than enough to run this

perhaps we can keep and improve the "dont do this at home" message

Reply:
trying to think if we can use an env var in ci and set a default here - ill test more tomorrow

Reply:
updated